src/Resource/Processor/CpuCoresCountProvider.php

```diff
@@ -65,24 +66,30 @@ public static function provide(): int
             return 1;
         }
 
-        // for Linux
-        $hasNproc = trim(@shell_exec('command -v nproc'));
+        try {
+            // for Linux
+            $hasNproc = trim(@shell_exec('command -v nproc'));
```

Comment:
one issue I see with this try/catch is that it's unclear which statement you really expect to throw

Reply:
Well the idea is mostly not to block i.e. Mac/Windows users when the Linux command does fail, to allow the next part of the function to run without trouble.

Reply:
I was more referring at the statement causing failure, here I doubt that anything apart from the shell exec is suppose to throw right?

Reply:
Right. Actually if any of the two `shell_exec` calls fails, it should be caught IMO, no matter which one. WDYT?